gapis/service/path/path.proto

```diff
@@ -44,6 +44,7 @@ message Any {
     FramebufferAttachments framebuffer_attachments = 18;
     FramebufferAttachment framebuffer_attachment = 19;
     Field field = 20;
+    Framegraph framegraph = 44;
```

Comment:
Style: I am not sure about this id. We update client and server at the same time. Therefore changing the number wouldn't be an issue(except the unnecessary change of 20 lines).

But outside of that we have a unusual usage of protobuff field numbers here. We put them in alphabetical order and chronological order(the numbers). As we easily think that we cannot guarantee that every new field will be continued to named alphabetically, just as in here, breaking this is inevitable. But, in my opinion, the more important one of those is the numbers instead of alphabetical order. So rather than breaking the number order, I would break the alphabetical order. Because every time we have to add a new field here, we would need to check all the numbers and ensure we don't have the same number. Because they have to be unique. But keeping the alphabetical order does not give anything to us.

Reply:
Yes, thanks for spotting this! I was trying to at the same time not rewrite IDs and keep the alphabetical order, but as you say that approach would lead to something where it's difficult to know the current max ID.

I've moved this to the bottom of the list.